react/components/ProductSummaryImage/ProductImage.js

```diff
@@ -172,7 +173,7 @@ const ProductImageContent = ({
 
   let skuImageUrl = pathOr('', ['image', 'imageUrl'], sku)
 
-  if (!skuImageUrl || hasError) {
+  if (!placeholder && (!skuImageUrl || hasError)) {
```

Comment:
This is more of a suggestion, but since you're going to perform the same check for `(!skuImageUrl || hasError)` a few lines down, you could store this result in a variable, maybe `shouldDisplayPlaceholder`, and then this conditional would also become clearer to read:

```js
if (shouldDisplayPlaceholder && !placeholder) {
```

Then in line 224 it would be:

```js
src={shouldDisplayPlaceholder ? placeholder : skuImageUrl}
```